### ISSUE TITLE
Update amMoment.changeLocale()

### ISSUE THREAD

**Issue by geroyche** [urish/angular-moment#273]
as per the current moment.js docs, moment.locale() is deprecated (and produces warnings in the console). it should be replaced with updateLocale()

https://github.com/moment/moment/issues/3043#issuecomment-197017584

**Comment by urish** [comment]
It seems like only the second parameter to `locale` was deprecated (i.e. customizing strings). Can you please send a PR that will call `updateLocale` if the second parameter is provided?

Thanks!

**Comment by geroyche** [comment]
not quite so. it's about whether the locale already exists or not.
.locale() if not, .updateLocale() if it does.
maybe it would be best to additionally provide .locale(9 and updateLocale() to your directive? that was existing code will continue to work, and in new code users can decide what to call, based on their knowledge about whether a locale exists already or not.

**Comment by urish** [comment]
Yes, that makes sense. I think that the best solution would be to deprecate the second parameter to `changeLocale`. If you want to define a new locale, or update an existing one - just go directly to `moment` API.
